endpoints/openrtb2/auction.go

```diff
@@ -1128,16 +1123,27 @@ func setUAImplicitly(httpReq *http.Request, bidReq *openrtb.BidRequest) {
 	}
 }
 
-// Check if a request comes from a Safari browser
-func checkSafari(r *http.Request) (isSafari bool) {
-	isSafari = false
+// parseUserId gets this user's ID  for the host machine, if it exists.
+func parseUserID(cfg *config.Configuration, httpReq *http.Request) (string, bool) {
+	if hostCookie, err := httpReq.Cookie(cfg.HostCookie.CookieName); hostCookie != nil && err == nil {
+		return hostCookie.Value, true
+	} else {
+		return "", false
+	}
+}
+
+// checkBrowserName checks if a request comes from a Safari browser.
+// Returns pbsmetrics.BrowserSafari or pbsmetrics.BrowserOther
+// depending on the value of the "User-Agent" header of our http.Request
+func checkBrowserName(r *http.Request) pbsmetrics.Browser {
```

Comment:
Consider renaming to something along the lines of `parseBrowser`. The "check" part of the name conveys an expectation for a boolean return, IMHO.

Reply:
Do you think `getBrowserName` works better? Or are you better with `parseBrowser`?

Reply:
`getBrowserName` sounds good to me. 👍

Reply:
Awesome. Changed